docs/testrunner-toolkit/configuration/cucumber.md

```diff
@@ -0,0 +1,21 @@
+---
+id: cucumber
+title: "Cucumber"
+sidebar_label: Cucumber
+---
+
+import Highlight from '../../../src/components/highlight.jsx'
```

Comment:
This import is no longer needed, as we don't use that highlight tag anymore.

Reply:
good to know, I can remove it.